src/pages/Integrations/Create/CreatePage.tsx

```diff
@@ -0,0 +1,108 @@
+import * as React from 'react';
+import { Messages } from '../../../properties/Messages';
+import {
+    Button,
+    Form,
+    FormGroup,
+    FormSelect,
+    FormSelectOption,
+    Modal,
+    TextInput
+} from '@patternfly/react-core';
+import { useEffect, useState } from 'react';
+import { IntegrationType } from "../../../types/Integration";
+
+const EMPTY = '';
+const title = Messages.components.integrations.toolbar.actions;
+const choose = Messages.common.choose;
+const webhookType = Messages.pages.integrations.types.hooks;
+
+const CreatePage = props => {
+    const [pageModal, updatePageModal] = useState({...props.isModalOpen});
+    const [newUrl, updateUrl] = useState("(enter hooks url)");
+    const [intCount,updateIntCount] = useState(0);
+    const handleTextFieldChanges = (evt) => {
+        updateUrl(evt);
+    };
+    const handleSubmit = () => {
+        if(newUrl){
+            updateIntCount(intCount+1);
+            props.updateModel(
+                ...props.model,{
+                id: 'newAddition-'+intCount,
+                isEnabled: true,
+                name: 'Pager duty'+intCount,
+                type: IntegrationType.HTTP,
+                url: newUrl
+            }
+            );
+        }
+    }
+    const handleExit = () => {
+        updatePageModal(!pageModal);
+        props.updateModal(!pageModal);
```

Comment:
The `props.updateModal(!pageModal);` is fine.

I think we can remove the `updatePageModal(!pageModal);` (along with the `const [pageModal, updatePageModal] = useState({...props.isModalOpen});`)